### DISCUSSION TITLE
What is the best hosting service for Next.js in the world?

### DISCUSSION THREAD

**Discussion by Pro-WebTech** [community/community#19972] [Other Feature Feedback, Questions, & Ideas] [answered]
I am using Vercel for that.
Is there other things better than Vercel?

**Comment by lucferrario** [comment]
Hi, for my personal experience i highly suggest you vercel. Just super easy and fast

**Comment by diegoleondev** [comment, marked as answer]
## Vercel is the best hosting for Next.js

Vercel has an integration and optimization for next js.

**Reply by Pro-WebTech** [reply]
Thanks.

**Comment by PrarthnaLuthra** [comment]
For Next.js applications **_prefer using Vercel as the Vercel platform itself is developed by the creators of Next.js._** You can also try Netlify

**Comment by inapeace0** [comment]
Interesting discussion!!! I think **Vercel** is the best platform for Next.js, and I recommend **Heroku** as well. Heroku has the great infrastructure and it's very suitable for SPA.
